src/openforms/conf/base.py

```diff
@@ -168,6 +168,7 @@
     "django.middleware.common.CommonMiddleware",
     "django.middleware.csrf.CsrfViewMiddleware",
     "django.contrib.auth.middleware.AuthenticationMiddleware",
+    "openforms.middleware.SessionTimeoutMiddleware",
```

Comment:
I put the middleware here since it needs to come after the `AuthenticationMiddleware` since that is what adds the user to the request so we can do `request.user`.

Ref: https://github.com/django/django/blob/main/django/contrib/auth/middleware.py#L25